CompileTests/MultiModule/Sources/ModuleA/a.pb.swift

```diff
@@ -75,7 +75,7 @@ public struct A: SwiftProtobuf.ExtensibleMessage, Sendable {
 
 // MARK: - Extension Properties
 
-// Swift Extensions on the exteneded Messages to add easy access to the declared
+// Swift Extensions on the extended Messages to add easy access to the declared
```

Comment:
This should be fixed in the a.proto file otherwise it will get lose on the next generation.